src/QMCTools/CMakeLists.txt

```diff
@@ -51,9 +53,15 @@ if(USE_OBJECT_TARGET)
   target_link_libraries(qmcfinitesize qmcparticle qmcutil)
 endif()
 
-install(TARGETS convert4qmc qmc-extract-eshdf-kvectors qmc-get-supercell qmc-check-affinity convertpw4qmc qmcfinitesize
+install(TARGETS convert4qmc qmc-extract-eshdf-kvectors qmc-get-supercell convertpw4qmc qmcfinitesize
```

Comment:
restore here.

Reply:
My bad, target install should not depend on `QMC_OMP`. Thanks!